devicetypes/widomsrl/widom-smart-dry-contact.src/widom-smart-dry-contact.groovy

```diff
@@ -0,0 +1,340 @@
+	logging("encapsulating command using MultiChannel Encapsulation, ep: $ep command: $cmd","info")
+	zwave.multiChannelV3.multiChannelCmdEncap(destinationEndPoint:ep).encapsulate(cmd)
+}
+
+private encap(physicalgraph.zwave.Command cmd, Integer ep) {
+	encap(multiEncap(cmd, ep))
+}
+
+private encap(physicalgraph.zwave.Command cmd) {
+	if (zwaveInfo.zw.contains("s")) {
+		secEncap(cmd)
+	} else if (zwaveInfo.cc.contains("56")){
+		crcEncap(cmd)
+	} else {
+		logging("no encapsulation supported for command: $cmd","info")
+		cmd.format()
+	}
+}
+
+private encapSequence(cmds, Integer delay=250) {
+	delayBetween(cmds.collect{ encap(it) }, delay)
+}
+
+private encapSequence(cmds, Integer delay, Integer ep) {
+	delayBetween(cmds.collect{ encap(it, ep) }, delay)
+}
+
+private List intToParam(Long value, Integer size = 1) {
+	def result = []
+	size.times {
+		result = result.plus(0, (value & 0xFF) as Short)
+		value = (value >> 8)
+	}
+	return result
+}
+
+/*
+##########################
+## Device Configuration ##
+##########################
+*/
+private Map cmdVersions() {
+	[0x5E: 2, 0x86: 2, 0x72: 2, 0x59: 2, 0x98: 1, 0x25: 1, 0x5A: 1, 0x85: 2, 0x70: 1, 0x8E: 2, 0x6C: 1] 
+}
+
+private parameterMap() {[
+		[key: "numClickLoad", num: 1, size: 1, type: "number", min: 0, max: 7, def: 7, title: "Numbers of clicks to controlthe loads",
+			descr: "Define which sequences of clicks control the load (see device manual)."],     
+		[key: "OffTimer", num: 10, size: 2, type: "number", def: 0, min: 0, max: 32000, title: " Timer to switch OFF the Relay",
+			descr: "Defines the time after which the relay is switched OFF. Time unit is set by parameter 15(see device manual)"],
+		[key: "OnTimer", num: 11, size: 2, type: "number", def: 0, min: 0, max: 32000, title: " Timer to switch ON the Relay",
+			descr: "Defines the time after which the relay is switched ON. Time unit is set by parameter 15(see device manual)"],
+		[key: "timerScale", num: 15, size: 1, type: "enum", options: [
+			1: "Tenth of seconds",
+			2: "Seconds",
+			], def: "1", title: "Timer scale", descr: "Defines the time unit used for parameters No.10 and No.11"],
+		[key: "oneClickScene", num: 20, size: 2, type: "number",min: 0, max: 255, def: 0, title: "One Click Scene ActivationSet",
+			descr: "Defines the Scene Activation Set value sent to the Lifeline group with 1 Clickon the external switch"],
+		[key: "twoClickScene", num: 21, size: 2, type: "number",min: 0, max: 255, def: 0, title: "Two Clicks Scene ActivationSet",
+			descr: "Defines the Scene Activation Set value sent to the Lifeline group with 2 Clickson the external switch"],
+		[key: "threeClickScene", num: 22, size: 2, type: "number",min: 0, max: 255, def: 0, title: "Three Clicks Scene ActivationSet",
+			descr: "Defines the Scene Activation Set value sent to the Lifeline group with 1 Clicks on the external switch"],
+		[key: "startUpStatus", num: 60, size: 1, type: "enum", options: [
+				1: "ON",
```

Comment:
Could You fix the indentation here ? (it seems that 3 tabs are enough here)

Reply:
fixed